src/routes/Landing.scss

```diff
@@ -86,6 +86,7 @@ sup.ins-c-rball { @include rem('font-size', 10px); }
         justify-content: center;
         text-align: center;
         &--body { width: 75%; }
+        &--footer-secondary { margin-top: 8px;}
```

Comment:
&--footer-secondary {var(--pf-global--spacer--sm);}

@arivepr Let's use the PF variables when possible
https://www.patternfly.org/v4/guidelines/spacers/

Reply:
woops, switched it out @epwinchell ! thanks